Add new stirmzi-view cluster role and configure aggregation to view, edit and admin roles (#2870)
strimzi/strimzi-kafka-operator: documentation/assemblies/assembly-getting-started-strimzi-admin.adoc

= Strimzi Administrators

> {ProductName} includes several custom resources.

Reviewer: _Strimzi provides custom resources for configuration of your deployment._

I moved and did some minor edits to this file for the deployment guide PR.
Will need to make sure the changes in this PR are reflected.

By default, permission to view, create, edit, and delete these resources is limited to Kubernetes cluster administrators.

> {ProductName} provides two additional `ClusterRoles` which you can use to assign these rights also to other users.

Reviewer: ```suggestion
{ProductName} provides two cluster roles, which you can use to assign these rights also to other users.
```

> The role `strimzi-view` allows used to view and list the {ProductName} resources.

Reviewer 2: "allows users" or "is used" ... there is a typo somewhere :-)

> The role `strimzi-view` allows used to view and list the {ProductName} resources.

Reviewer: ```suggestion
`strimzi-view` allows users to view and list {ProductName} resources.
```

> The role `strimzi-admin` allows them to also create, edit or delete them.

Reviewer 2: "them" is used twice here with different meanings, maybe it's better to specify "users" and "{ProductName} resources" ?

> The role `strimzi-admin` allows them to also create, edit or delete them.

Reviewer: ```suggestion
`strimzi-admin` allows users to also create, edit or delete {ProductName} resources.
```

> When you install these roles, they will automatically add (aggregate) these rights to the default Kubernetes `ClusterRoles`.

Reviewer: ```suggestion
When you install these roles, they will automatically add (aggregate) these rights to the default Kubernetes cluster roles.
```

> `strimzi-view` will aggregate to the `view` role and `strimzi-admin` to the `edit` and `admin` roles.

Reviewer: ```suggestion
`strimzi-view` aggregates to the `view` role, and `strimzi-admin` aggregates to the `edit` and `admin` roles.
```

> Thanks to the aggregation, you might not need to assign these roles to users who already have similar rights.

Reviewer: ```suggestion
Because of the aggregation, you might not need to assign these roles to users who already have similar rights.
```
